app/assess/data.py

```diff
@@ -151,6 +151,17 @@ def get_round(
     return None
 
 
+def get_available_teams(fund_id: str, round_id: str) -> list:
+    # TODO: Rework on the avialable teams after implemented in fundstore
+    teams_available = get_data(
+        Config.GET_AVIALABLE_TEAMS_FOR_FUND.format(
+            fund_id=fund_id,
+            round_id=round_id,
+        )
+    )
+    return [] if not teams_available else teams_available
```

Comment:
Alternative, but your solution also works and not a big deal:

```suggestion
    return teams_available or []
```